diff --git a/node_modules/react-native-material-buttons/src/components/button/index.js b/node_modules/react-native-material-buttons/src/components/button/index.js
index d3d9088..aac9ac4 100644
--- a/node_modules/react-native-material-buttons/src/components/button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/button/index.js
@@ -25,24 +25,24 @@ export default class Button extends PureComponent {
     disabled: false,
   };
 
-  static propTypes = {
-    ...Ripple.propTypes,
+  // static propTypes = {
+  //   ...Ripple.propTypes,
 
-    color: PropTypes.string,
-    disabledColor: PropTypes.string,
+  //   color: PropTypes.string,
+  //   disabledColor: PropTypes.string,
 
-    shadeColor: PropTypes.string,
-    shadeOpacity: PropTypes.number,
-    shadeBorderRadius: PropTypes.number,
+  //   shadeColor: PropTypes.string,
+  //   shadeOpacity: PropTypes.number,
+  //   shadeBorderRadius: PropTypes.number,
 
-    focusAnimation: PropTypes.instanceOf(Animated.Value),
-    focusAnimationDuration: PropTypes.number,
+  //   focusAnimation: PropTypes.instanceOf(Animated.Value),
+  //   focusAnimationDuration: PropTypes.number,
 
-    disableAnimation: PropTypes.instanceOf(Animated.Value),
-    disableAnimationDuration: PropTypes.number,
+  //   disableAnimation: PropTypes.instanceOf(Animated.Value),
+  //   disableAnimationDuration: PropTypes.number,
 
-    payload: PropTypes.any,
-  };
+  //   payload: PropTypes.any,
+  // };
 
   constructor(props) {
     super(props);
diff --git a/node_modules/react-native-material-buttons/src/components/raised-button/index.js b/node_modules/react-native-material-buttons/src/components/raised-button/index.js
index 47ce59c..a77e76b 100644
--- a/node_modules/react-native-material-buttons/src/components/raised-button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/raised-button/index.js
@@ -5,9 +5,9 @@ import { styles } from './styles';
 import Button from '../button';
 
 export default class RaisedButton extends PureComponent {
-  static propTypes = {
-    ...Button.propTypes,
-  };
+  // static propTypes = {
+  //   ...Button.propTypes,
+  // };
 
   constructor(props) {
     super(props);
diff --git a/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js b/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
index 22ac30c..fec67ab 100644
--- a/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/raised-text-button/index.js
@@ -11,14 +11,14 @@ export default class RaisedTextButton extends PureComponent {
     disabledTitleColor: 'rgba(0, 0, 0, .26)',
   };
 
-  static propTypes = {
-    ...RaisedButton.propTypes,
+  // static propTypes = {
+  //   ...RaisedButton.propTypes,
 
-    title: PropTypes.string.isRequired,
-    titleColor: PropTypes.string,
-    titleStyle: Animated.Text.propTypes.style,
-    disabledTitleColor: PropTypes.string,
-  };
+  //   title: PropTypes.string.isRequired,
+  //   titleColor: PropTypes.string,
+  //   titleStyle: Animated.Text.propTypes.style,
+  //   disabledTitleColor: PropTypes.string,
+  // };
 
   constructor(props) {
     super(props);
diff --git a/node_modules/react-native-material-buttons/src/components/text-button/index.js b/node_modules/react-native-material-buttons/src/components/text-button/index.js
index e8c13b6..9b90a03 100644
--- a/node_modules/react-native-material-buttons/src/components/text-button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/text-button/index.js
@@ -16,14 +16,14 @@ export default class TextButton extends PureComponent {
     shadeOpacity: 0.20,
   };
 
-  static propTypes = {
-    ...Button.propTypes,
+  // static propTypes = {
+  //   ...Button.propTypes,
 
-    title: PropTypes.string.isRequired,
-    titleColor: PropTypes.string,
-    titleStyle: Animated.Text.propTypes.style,
-    disabledTitleColor: PropTypes.string,
-  };
+  //   title: PropTypes.string.isRequired,
+  //   titleColor: PropTypes.string,
+  //   titleStyle: Animated.Text.propTypes.style,
+  //   disabledTitleColor: PropTypes.string,
+  // };
 
   constructor(props) {
     super(props);
