diff --git a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
index b1e65a1..cc459e2 100644
--- a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
+++ b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
@@ -12,6 +12,7 @@ import {
   ViewPropTypes,
   I18nManager,
 } from 'react-native';
+import { TextPropTypes } from 'deprecated-react-native-prop-types';
 import Ripple from 'react-native-material-ripple';
 import { TextField } from 'react-native-material-textfield';
 
@@ -78,82 +79,82 @@ export default class Dropdown extends PureComponent {
     useNativeDriver: false,
   };
 
-  static propTypes = {
-    ...TouchableWithoutFeedback.propTypes,
+  // static propTypes = {
+  //   ...TouchableWithoutFeedback.propTypes,
 
-    disabled: PropTypes.bool,
+  //   disabled: PropTypes.bool,
 
-    value: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.number,
-    ]),
+  //   value: PropTypes.oneOfType([
+  //     PropTypes.string,
+  //     PropTypes.number,
+  //   ]),
 
-    data: PropTypes.arrayOf(PropTypes.object),
+  //   data: PropTypes.arrayOf(PropTypes.object),
 
-    valueExtractor: PropTypes.func,
-    labelExtractor: PropTypes.func,
-    propsExtractor: PropTypes.func,
+  //   valueExtractor: PropTypes.func,
+  //   labelExtractor: PropTypes.func,
+  //   propsExtractor: PropTypes.func,
 
-    absoluteRTLLayout: PropTypes.bool,
+  //   absoluteRTLLayout: PropTypes.bool,
 
-    dropdownOffset: PropTypes.shape({
-      top: PropTypes.number.isRequired,
-      left: PropTypes.number.isRequired,
-    }),
+  //   dropdownOffset: PropTypes.shape({
+  //     top: PropTypes.number.isRequired,
+  //     left: PropTypes.number.isRequired,
+  //   }),
 
-    dropdownMargins: PropTypes.shape({
-      min: PropTypes.number.isRequired,
-      max: PropTypes.number.isRequired,
-    }),
+  //   dropdownMargins: PropTypes.shape({
+  //     min: PropTypes.number.isRequired,
+  //     max: PropTypes.number.isRequired,
+  //   }),
 
-    dropdownPosition: PropTypes.number,
+  //   dropdownPosition: PropTypes.number,
 
-    rippleColor: PropTypes.string,
-    rippleCentered: PropTypes.bool,
-    rippleSequential: PropTypes.bool,
+  //   rippleColor: PropTypes.string,
+  //   rippleCentered: PropTypes.bool,
+  //   rippleSequential: PropTypes.bool,
 
-    rippleInsets: PropTypes.shape({
-      top: PropTypes.number,
-      right: PropTypes.number,
-      bottom: PropTypes.number,
-      left: PropTypes.number,
-    }),
+  //   rippleInsets: PropTypes.shape({
+  //     top: PropTypes.number,
+  //     right: PropTypes.number,
+  //     bottom: PropTypes.number,
+  //     left: PropTypes.number,
+  //   }),
 
-    rippleOpacity: PropTypes.number,
-    shadeOpacity: PropTypes.number,
+  //   rippleOpacity: PropTypes.number,
+  //   shadeOpacity: PropTypes.number,
 
-    rippleDuration: PropTypes.number,
-    animationDuration: PropTypes.number,
+  //   rippleDuration: PropTypes.number,
+  //   animationDuration: PropTypes.number,
 
-    fontSize: PropTypes.number,
+  //   fontSize: PropTypes.number,
 
-    textColor: PropTypes.string,
-    itemColor: PropTypes.string,
-    selectedItemColor: PropTypes.string,
-    disabledItemColor: PropTypes.string,
-    baseColor: PropTypes.string,
+  //   textColor: PropTypes.string,
+  //   itemColor: PropTypes.string,
+  //   selectedItemColor: PropTypes.string,
+  //   disabledItemColor: PropTypes.string,
+  //   baseColor: PropTypes.string,
 
-    itemTextStyle: Text.propTypes.style,
+  //   itemTextStyle: TextPropTypes.style,
 
-    itemCount: PropTypes.number,
-    itemPadding: PropTypes.number,
+  //   itemCount: PropTypes.number,
+  //   itemPadding: PropTypes.number,
 
-    onLayout: PropTypes.func,
-    onFocus: PropTypes.func,
-    onBlur: PropTypes.func,
-    onChangeText: PropTypes.func,
+  //   onLayout: PropTypes.func,
+  //   onFocus: PropTypes.func,
+  //   onBlur: PropTypes.func,
+  //   onChangeText: PropTypes.func,
 
-    renderBase: PropTypes.func,
-    renderAccessory: PropTypes.func,
+  //   renderBase: PropTypes.func,
+  //   renderAccessory: PropTypes.func,
 
-    containerStyle: (ViewPropTypes || View.propTypes).style,
-    overlayStyle: (ViewPropTypes || View.propTypes).style,
-    pickerStyle: (ViewPropTypes || View.propTypes).style,
+  //   containerStyle: (ViewPropTypes || View.propTypes).style,
+  //   overlayStyle: (ViewPropTypes || View.propTypes).style,
+  //   pickerStyle: (ViewPropTypes || View.propTypes).style,
 
-    supportedOrientations: PropTypes.arrayOf(PropTypes.string),
+  //   supportedOrientations: PropTypes.arrayOf(PropTypes.string),
 
-    useNativeDriver: PropTypes.bool,
-  };
+  //   useNativeDriver: PropTypes.bool,
+  // };
 
   constructor(props) {
     super(props);
diff --git a/node_modules/react-native-material-dropdown/src/components/item/index.js b/node_modules/react-native-material-dropdown/src/components/item/index.js
index 22e3635..87838cc 100644
--- a/node_modules/react-native-material-dropdown/src/components/item/index.js
+++ b/node_modules/react-native-material-dropdown/src/components/item/index.js
@@ -12,11 +12,11 @@ export default class DropdownItem extends PureComponent {
     shadeBorderRadius: 0,
   };
 
-  static propTypes = {
-    ...Button.propTypes,
+  // static propTypes = {
+  //   ...Button.propTypes,
 
-    index: PropTypes.number.isRequired,
-  };
+  //   index: PropTypes.number.isRequired,
+  // };
 
   constructor(props) {
     super(props);
